src/inventory/inventory.resolver.ts

```diff
@@ -34,7 +38,7 @@ export class InventoryResolver {
     @Args('createProductItemBatchInput')
     createProductItemBatchInput: CreateProductItemBatchInput,
   ) {
-    console.log('Resolving createProductItemBatch for ', createProductItemBatchInput)
+    this.logger.debug(`Resolving createProductItemBatch for ${JSON.stringify(createProductItemBatchInput)}`)
```

Comment:
Additionally, does winston provide a way to only evaluate the parameters once it knows that the log will recorded?

Reply:
Could you elaborate on that? On The log levels are connected to transports, debug print to console, all other levels to console and file

Reply:
What I mean: typically, loggers offer a syntax of the form `log(message, parameters)`, so that the message is only constructed based on the parameters when actually executed, not always.
But that's not a blocking aspect here.